### DISCUSSION TITLE
Support Policy？

### DISCUSSION THREAD

**Discussion by Varorbc** [microsoft/aspire#6438] [General]
Hi there! I'm trying to get some info on the support policy for .NET Aspire. I've been looking through the relevant documents but couldn't find any mention of it. Did I miss something? If you could point me in the right direction or share the correct link, that would be great!

**Comment by davidfowl** [comment]
https://dotnet.microsoft.com/en-us/platform/support/policy/aspire

**Reply by Varorbc** [reply]
Thanks a million! It would be great if you could mention the support policy on the .NET Aspire documentation page or in the README on GitHub.

**Reply by davidfowl** [reply]
https://github.com/dotnet/docs-aspire/issues/1882
